packages/analytics-v1.1/src/core/analytics.js

```diff
@@ -603,30 +607,34 @@ class Analytics {
 
     // send the queued events to the fetched integration
     object.toBeProcessedByIntegrationArray.forEach(event => {
```

Comment:
Move Ln 606 to the beginning of this function so that delivery to the data plane is unaffected by the destinations.

Reply:
Aren't CM events dependent on the integration object construction? which is happening on line no 597

Reply:
You are probably right. So you can isolate that above block with error handling.

Reply:
Addressed